test/integration/integration.cc

```diff
@@ -212,8 +211,8 @@ void IntegrationTcpClient::ConnectionCallbacks::onEvent(uint32_t events) {
 
 BaseIntegrationTest::BaseIntegrationTest()
     : api_(new Api::Impl(std::chrono::milliseconds(10000))),
-      dispatcher_(api_->allocateDispatcher()) {
-
+      dispatcher_(api_->allocateDispatcher()),
+      default_log_level_(TestEnvironment::getOptions().logLevel()) {
```

Comment:
I didn't want `//test:main` to take a dependency on `//test/integration:integration_lib`, so rather than initialize the library in `main.cc`, we just stash away the args for later when used by integration tests. This then avoids building `integration_lib` (and consequently, most of Envoy), when running tiny tests like `//test/common/common:hex_test`.